jerry-core/vm/vm.c

```diff
@@ -2304,24 +2310,38 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           *stack_top_p++ = result;
           continue;
         }
-        case VM_OC_GET_ITERATOR:
+        case VM_OC_ITERATOR_CONTEXT_CREATE:
         {
-          result = ecma_op_get_iterator (stack_top_p[-1], ECMA_VALUE_SYNC_ITERATOR, NULL);
+          result = ecma_op_get_iterator (stack_top_p[-1], ECMA_VALUE_SYNC_ITERATOR, &left_value);
 
           if (ECMA_IS_VALUE_ERROR (result))
           {
             goto error;
           }
 
-          *stack_top_p++ = result;
+          uint32_t context_size = (uint32_t) (stack_top_p + PARSER_ITERATOR_CONTEXT_STACK_ALLOCATION
+                                              - VM_LAST_CONTEXT_END ());
```

Comment:
There are a + and - operators, so this is a style error. It does not look too nice.